doc/src/manual/types.md

```diff
@@ -634,20 +631,16 @@ Point{Float64}
 
 For the default constructor, exactly one argument must be supplied for each field:
 
-```julia
+```jldoctest pointtype
 julia> Point{Float64}(1.0)
 ERROR: MethodError: Cannot `convert` an object of type Float64 to an object of type Point{Float64}
 This may have arisen from a call to the constructor Point{Float64}(...),
 since type constructors fall back to convert methods.
- in Point{Float64}(::Float64) at ./sysimg.jl:66
- ...
+Stacktrace:
+ [1] Point{Float64}(::Float64) at ./sysimg.jl:24
 
 julia> Point{Float64}(1.0,2.0,3.0)
 ERROR: MethodError: no method matching Point{Float64}(::Float64, ::Float64, ::Float64)
-Closest candidates are:
-  Point{Float64}{T}(::Any, ::Any) at none:3
-  Point{Float64}{T}(::Any) at sysimg.jl:66
```

Comment:
IIRC there were none shown for this example. Note that there are no `[...]` added.